scripts/did.sh

```diff
@@ -2,15 +2,15 @@
 
 function generate_did() {
   local canister=$1
-  canister_root="src/$canister"
+  local candid_file="$(canister="$canister" jq -r .canisters[env.canister].candid dfx.json)"
 
   test -e "target/wasm32-unknown-unknown/release/$canister.wasm" ||
-    cargo build --manifest-path="$canister_root/Cargo.toml" \
+    cargo build \
```

Comment:
The path to the `Cargo.toml` is provided by the workspace, so does not need to be provided explicitly.